AdobeStockImage/Model/SaveMediaGalleryAsset.php

```diff
@@ -86,4 +89,19 @@ private function calculateFileSize(string $destinationPath): int
         $mediaDirectory = $this->fileSystem->getDirectoryRead(DirectoryList::MEDIA);
         return $mediaDirectory->stat($mediaDirectory->getAbsolutePath($destinationPath))['size'];
     }
+
+    /**
+     * Hash image content.
+     *
+     * @param string $destinationPath
+     * @return string
+     * @throws FileSystemException
+     */
+    private function hashImageContent(string $destinationPath): string
+    {
+        $mediaDirectory = $this->fileSystem->getDirectoryRead(DirectoryList::MEDIA);
+        $imageContent = $mediaDirectory->readFile($mediaDirectory->getAbsolutePath($destinationPath));
+        $hashedImageContent = sha1($imageContent);
+        return $hashedImageContent;
+    }
```

Comment:
```suggestion
    private function hashImageContent(string $destinationPath): string
    {
        return sha1_file($destinationPath);
    }
```
cc @sivaschenko

Reply:
Read the comment https://github.com/magento/adobe-stock-integration/pull/1510#discussion_r444850084 and see the point with the `sha1_file` @joweecaquicla please, ignore my change request.